### PR TITLE
Feature/html

### LINKED ISSUE #6: Recipe Repo/Recipe/Ingredient Class
### Recipe Repository Class: 
- takes recipe object as param
- 2 methods:
- [x] filter recipe by tag
- [x] filter recipe by name || ingredient
### Recipe Class:
- takes one recipe object
- 3 methods:
- [ ] sort/every/filter ingred. needed
- [ ] calculate cost ingredients
- [ ] find/return instructions/directions
### Ingredient Class:
- To build recipes from

### LINKED ISSUE #8: Readability
- [x] Spacing between class methods for readability
- [x] File names aren't capitalized to represent that they're classes (user.js should be User.js)

### LINKED ISSUE #19: USER CLASS REFACTOR
* Iteration Two Methods: 
- [x] favorite and unfavorite recipes
(add to/ remove from `users favorite recipes`
- [ ] decide to cook a recipe per week
(add to `recipesToCook`)
- [ ] filter `favoriteRecipes` by one or more tags
- [ ] filter `favoriteRecipes` by name or ingredient
- [x] Moved filterRecipe methods to `RecipeRepository` class

### PR DESCRIPTION
**Pull request by rhen92**
# Description of Changes

- Change cardHTML
- Add .text:focus-visible
- Add aria-expanded to pantry button
- Data is now being pulled from fetch calls
- Add RecipeRepository class
- Refactor search functions to incorporate methods from RecipeRepository

# Linked Issues

- Closes #8, Closes #6, Closes #19

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,32 +1,27 @@
 import User from './user';
 
-let users;
-let ingredientData;
-let recipeData;
+let users = [];
```
Comment (battan40) [inline src/apiCalls.js:3]: This was a great solution, lets review together to verify its best practice

**Review by battan40 — APPROVED** [review]

File: src/css/styles.scss [hunk 2]
```diff
@@ -131,7 +131,7 @@ header {
   font-size: 0px;
   font-weight: 700;
   justify-content: center;
-  outline: none;
+  // outline: none;
```
Comment (mistercanderson) [inline src/css/styles.scss:134]: This was a great find, like a needle in a haystack for accessibility

File: src/scripts.js [hunk 3]
```diff
@@ -331,18 +331,16 @@ function filterNonSearched(filtered) {
...
   var menuDropdown = document.querySelector('.drop-menu');
+  let attr = buttons.pantry.getAttribute("aria-expanded");
   pantryMenuOpen = !pantryMenuOpen;
-  if (pantryMenuOpen) {
+  if (pantryMenuOpen && attr === 'false') {
     menuDropdown.style.display = 'block';
+    buttons.pantry.setAttribute("aria-expanded", true);
   } else {
     menuDropdown.style.display = 'none';
+    buttons.pantry.setAttribute("aria-expanded", false);
```
Comment (mistercanderson) [inline src/scripts.js:343]: This is another really great accessibility feature

**Review by mistercanderson — COMMENTED** [review]
This looks great!

**Pull request merged**